src/libs/ReportUtils.ts

```diff
@@ -955,6 +955,14 @@ function isProcessingReport(report: OnyxEntry<Report> | EmptyObject): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report?.statusNum === CONST.REPORT.STATUS_NUM.SUBMITTED;
 }
 
+/**
+ * Returns true if the policy is of `instant` submit type and if the report is still being processed (i.e. submitted)
+ */
+function isInstantSubmittedState(report: OnyxEntry<Report> | EmptyObject): boolean {
```

Comment:
NAB - Or `isInInstantSubmittedState`? 🤔  I don't like it, but can't think of something else better...

Maybe `isExpenseReportWithInstantSubmittedState` but that's quite long?

Reply:
`isExpenseReportWithInstantSubmittedState` may be long but I think it conveys the message well. Can I go ahead and use this? 
cc @Beamanator

Reply:
I think yeah I prefer the long name 👍  @Ollyws any thoughts here before we update?

Reply:
Nice, ok let's go with this new name 👍

Reply:
Committed changes to use the new name.